src/main/java/cuchaz/enigma/EnigmaProfile.java

```diff
@@ -33,7 +33,7 @@ public static EnigmaProfile parse(Reader reader) {
 	}
 
 	@Nullable
-	public Service getServiceProfile(EnigmaServiceType<?> serviceType) {
+	public Service[] getServiceProfile(EnigmaServiceType<?> serviceType) {
```

Comment:
Should rename to be plural now probably like `getServiceProfiles`

Reply:
Fixed.